nvm.sh

```diff
@@ -2093,6 +2093,14 @@ nvm_get_download_slug() {
     fi
   fi
 
+  # If node version in below 16.0.0 then there is no arm64 packages available in nod erepositories, so we have to install "x64" arch packages
+  # If running MAC M1 :: arm64 ARCH and Darwin OS then use "x64" Architecture because node doesn't provide darwin_arm64 package
+  if ! nvm_version_greater_than_or_equal_to "${VERSION}" 16.0.0; then
```

Comment:
```suggestion
  if nvm_version_greater_than_or_equal_to '16.0.0' "${VERSION}"; then
```
i think this needs to be inverted so that 16.0.0 is included? tricky to be sure without testing it.

Reply:
Thanks for the suggestions in code formatting and coding style. Really means a lot to me. I have changed readme statements after reverting changes.